src/Jackett.Common/Definitions/metal-iplay-ro.yml

```diff
@@ -0,0 +1,83 @@
+---
+  site: metaliplayro
+  name: metal.iplay.ro
+  description: "Romanian Metal Torrent is a Semi-Private site dedicated to METAL MUSIC. This definition is for the English site."
+  language: en-us
+  type: semi-private
+  encoding: UTF-8
+  links:
+    - http://metal.iplay.ro/
+
+  caps:
+    categories:
+      "Music": Audio
+
+    modes:
+      search: [q]
+      music-search: [q, album, artist, label, year]
+
+  login:
+    path: login.php
+    method: form
+    form: form[action="takelogin.php"]
+    captcha:
+      type: image
+      selector: img
+      input: vImageCodP
+    inputs:
+      username: "{{ .Config.username }}"
+      password: "{{ .Config.password }}"
+    error:
+      - selector: div.errorSummary
+        message:
+          selector: div.errorSummary ul li
+    test:
+      path: browse.php
+      selector: a[href="logout.php"]
+  
+  search:
+    paths:
+      - path: browse.php
+        method: get
+    inputs:
+      "search": "{{if .Query.Artist}}{{ .Query.Artist }}{{else}}{{ .Keywords }}{{end}}"
+    rows:
+      selector: table.torrents_table tbody tr
+    fields:
+      title:
+        selector: td a[href^="details.php?id="]
+      details:
+        selector: td a[href^="details.php?id="]
+        attribute: href
+      download:
+        selector: td a[href^="download2.php?id="]
+        attribute: href
+      banner: # link to a banner image (cover, etc.)
+        optional: true # don't fail if the selector doesn't match
+        selector: td:nth-child(2) a
+        attribute: onmouseover
+        filters:
+          - name: regexp
+            args: src=\\'(.+?)\\'
+          - name: replace # replace dummy image with empty string
+            args: ["./pic/noposter.jpg", ""]
+      date:
+        selector: td:nth-child(8) > span
+        attribute: title
+      size:
+        selector: td:nth-child(6)
+      seeders:
+        selector: td:nth-child(7)
+        filters:
+          - name: regexp
+            args: '([\d]+) seeders' # get the first number from the result
+      leechers:
+        selector: td:nth-child(7)
+        filters:
+          - name: regexp
+            args: '([\d]+) leechers' # get the first number from the result
+      grabs: # number of downloads
+        selector: td:nth-child(6)
+        filters:
+          - name: regexp
+            args: '([\d]+) times' # get the first number from the result
```

Comment:
might as well add a hard-coded category in the results if you are not going to process a selector, and a few other default fields
```yaml
      category:
        text: "Music"
      downloadvolumefactor:
        text: "0"
      uploadvolumefactor:
        text: "1"
```
as for the download, looks like they sabotaged the html with an extra quote at the end, to make the href invalid
`<a href="download2.php?id=104687"">`
so you are going to have to either fix it with a filter or build a custom one by fishing the id out of it

Reply:
oops, sorry, nothing wrong with your leechers, just bad eye sight on my part ;-)